docs/lib/datastore/fragments/android/relational.md

```diff
@@ -28,48 +28,54 @@ type Comment @model {
 ```java
 Post post = Post.builder()
     .title("My Post with comments")
-    .rating(10)
     .status(PostStatus.ACTIVE)
     .build();
 
-Amplify.DataStore.save(post, new ResultListener<DataStoreItemChange<Post>>() {
-    /* callback implementation */
-});
+Amplify.DataStore.save(post,
+    postSaved -> Log.i("AmplifyGetStarted", "Post saved"),
+    dataStoreException -> Log.e("AmplifyGetStarted", dataStoreException.getMessage(), dataStoreException)
```

Comment:
The log call will print out the message by virtue of the third argument. So, the way you're using the second param is duplicative. It's better to add a custom message that is local to the current code segment. Here, maybe just `"Failed to save a post."`.

^^ (Same for all uses of the three-arg `Log.xxx` family.)